.github/containers/ubuntu-miniforge-esm_tests/README.md

```diff
@@ -0,0 +1,19 @@
+To build the container from Mac OS:
+
+```bash
+docker buildx build --platform linux/x86-64 -t ubuntu-miniforge-esm_tests .
+```
+
+To run and test the container:
+
+```bash
+docker run -it --rm ubuntu-miniforge-esm_tests
+```
+
+To tag it and push it to Docker Hub:
+
+```bash
+docker tag ubuntu-miniforge-esm_tests mandresm/ubuntu-miniforge-esm_tests:v1.0
+docker login
+docker push mandresm/ubuntu-miniforge-esm_tests:v1.0
+```
```

Comment:
Not urgent, but GitHub has it's own internal container registry: [ghcr.io](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry), we could host the test container there.

Reply:
good to know